py3status/modules/battery_level.py

```diff
@@ -74,18 +81,21 @@
         (default None)
 
 Requires:
-    - the `acpi` command line
+    - the `acpi` the acpi command line utility (only if
+    `measurement_mode='acpi'`)
```

Comment:
```
2016-10-17 10:40:07 WARNING Setup error (FileNotFoundError) docstrings.py line 26.
2016-10-17 10:40:07 INFO Traceback
FileNotFoundError: [Errno 2] Arquivo ou diretório não encontrado: '/home/m45t3r/Source/py3status/env/lib/python3.5/site-packages/py3status-3.3rc0-py3.5.egg/py3status/modules/README.md'
  File "/home/m45t3r/Source/py3status/env/lib/python3.5/site-packages/py3status-3.3rc0-py3.5.egg/py3status/__init__.py", line 23, in main
    py3.setup()
  File "/home/m45t3r/Source/py3status/env/lib/python3.5/site-packages/py3status-3.3rc0-py3.5.egg/py3status/core.py", line 277, in setup
    self.handle_cli_command(self.config)
  File "/home/m45t3r/Source/py3status/env/lib/python3.5/site-packages/py3status-3.3rc0-py3.5.egg/py3status/core.py", line 787, in handle_cli_command
    docstrings.update_readme_for_modules(cmd[2:])
  File "/home/m45t3r/Source/py3status/env/lib/python3.5/site-packages/py3status-3.3rc0-py3.5.egg/py3status/docstrings.py", line 331, in update_readme_for_modules
    readme = parse_readme()
  File "/home/m45t3r/Source/py3status/env/lib/python3.5/site-packages/py3status-3.3rc0-py3.5.egg/py3status/docstrings.py", line 26, in parse_readme
    with open(readme_file) as f:

2016-10-17 10:40:07 ERROR py3status: Setup error (FileNotFoundError) docstrings.py line 26. Please try to fix this and reload i3wm (Mod+Shift+R)
```

Seems it is append my virtualenv isntead of looking at the relative path of the file.